main.py

```diff
@@ -0,0 +1,79 @@
+#   Domato - main script
```

Comment:
I thought about this some more and I actually prefer to have generator.py serve as main. Please merge it back.